tests/integration/pybaseball/test_statcast_pitcher.py

```diff
@@ -1,102 +1,102 @@
+    assert not result.empty
 
-	assert len(result.columns) == 18
-	assert len(result) == 428
+    assert len(result.columns) == 18
+    assert len(result) == 428
 
 def test_statcast_pitcher_pitch_arsenal() -> None:
-	minP = 250
-	result: pd.DataFrame = statcast_pitcher_pitch_arsenal(2019, minP=minP)
+    min_p = 250
+    result: pd.DataFrame = statcast_pitcher_pitch_arsenal(2019, min_p)
 
-	assert result is not None
-	assert not result.empty
+    assert result is not None
+    assert not result.empty
 
-	assert len(result.columns) == 11
-	assert len(result) == 609
+    assert len(result.columns) == 11
+    assert len(result) == 609
 
 def test_statcast_pitcher_arsenal_stats() -> None:
-	minPA = 25
-	result: pd.DataFrame = statcast_pitcher_arsenal_stats(2019, minPA=minPA)
+    min_pa = 25
+    result: pd.DataFrame = statcast_pitcher_arsenal_stats(2019, min_pa)
 
-	assert result is not None
-	assert not result.empty
+    assert result is not None
+    assert not result.empty
 
-	assert len(result.columns) == 21
-	assert len(result) == 1769
+    assert len(result.columns) == 21
+    assert len(result) == 1769
 
 def test_statcast_pitcher_pitch_movement() -> None:
-	minP = 250
-	result: pd.DataFrame = statcast_pitcher_pitch_movement(2019, minP=minP)
+    min_p = 250
+    result: pd.DataFrame = statcast_pitcher_pitch_movement(2019, min_p)
 
-	assert result is not None
-	assert not result.empty
+    assert result is not None
+    assert not result.empty
 
-	assert len(result.columns) == 24
-	assert len(result) == 347
+    assert len(result.columns) == 24
+    assert len(result) == 347
 
 def test_statcast_pitcher_active_spin() -> None:
-	minP = 250
-	result: pd.DataFrame = statcast_pitcher_active_spin(2019, minP=minP)
+    min_p = 250
+    result: pd.DataFrame = statcast_pitcher_active_spin(2019, min_p)
 
-	assert result is not None
-	assert not result.empty
+    assert result is not None
+    assert not result.empty
 
-	assert len(result.columns) == 10
-	assert len(result) == 606
+    assert len(result.columns) == 10
+    assert len(result) == 601
 
 def test_statcast_pitcher_percentile_ranks() -> None:
-	result: pd.DataFrame = statcast_pitcher_percentile_ranks(2019)
+    result: pd.DataFrame = statcast_pitcher_percentile_ranks(2019)
 
-	assert result is not None
-	assert not result.empty
+    assert result is not None
+    assert not result.empty
 
-	assert len(result.columns) == 19
-	assert len(result) == 831
+    assert len(result.columns) == 19
+    assert len(result) == 831
 
 def test_statcast_pitcher_spin_dir_comp() -> None:
-	result: pd.DataFrame = statcast_pitcher_spin_dir_comp(2020)
+    result: pd.DataFrame = statcast_pitcher_spin_dir_comp(2020)
 
-	assert result is not None
-	assert not result.empty
+    assert result is not None
+    assert not result.empty
 
-	assert len(result.columns) == 30
-	assert len(result) == 260
\ No newline at end of file
+    assert len(result.columns) == 30
+    assert len(result) == 260
```

Comment:
Is this necessary? (_sighs about formatter again_)

Reply:
Actually this was me again. Necessary? Not 100% (and I can revert it), I've just seen too many issues from having a mix of tabs and spaces in a project, especially with multiple participants working in multiple environments.